caasp-istio-proxyv2-image/caasp-istio-proxyv2-image.kiwi

```diff
@@ -30,6 +30,11 @@
             <label name="com.suse.reference" value="registry.suse.com/caasp/v5/istio-proxyv2:%%PKG_VERSION%%"/>
           </suse_label_helper:add_prefix>
         </labels>
+        <environment>
+          <env name="ISTIO_META_INTERCEPTION_MODE" value="TPROXY"/>
+          <env name="ISTIO_META_ISTIO_VERSION" value="%%PKG_VERSION%%"/>
```

Comment:
Note that this `%%PKG_VERSION%%` will be in the form of MAJOR.MINOR.PATCH in this case `1.5.4`, not sure if the package version is longer (some pkgs also include revision numbers)

Reply:
Yes, it's exactly what I need :)